test/routers/start.int.ts

```diff
@@ -1,39 +1,80 @@
 
 describe("Start router/handler integration tests", () => {
     beforeEach(() => {
         jest.clearAllMocks();
     });
 
     afterEach(() => {
-        expect(middlewareMocks.mockSessionMiddleware).toHaveBeenCalledTimes(1);
-        expect(middlewareMocks.mockFetchCompanyMiddleware).not.toHaveBeenCalled();
-        expect(middlewareMocks.mockFetchVerificationMiddleware).not.toHaveBeenCalled();
+        expect(mockSessionMiddleware).toHaveBeenCalledTimes(1);
+        expect(mockFetchCompanyMiddleware).not.toHaveBeenCalled();
+        expect(mockFetchVerificationMiddleware).not.toHaveBeenCalled();
     });
 
-    const HEADING = "Provide identity verification details for a person with significant control";
-
-    describe("GET method", () => {
+    const START_HEADING = "Provide identity verification details for a person with significant control";
+    const SERVICE_UNAVAILABLE_HEADING = "Sorry, the service is unavailable";
 
+    describe("GET method when no Planned Maintenance set", () => {
         it("should render the start page for service root URL", async () => {
             const resp = await request(app).get(servicePathPrefix);
 
             expect(resp.status).toBe(HttpStatusCode.Ok);
             const $ = cheerio.load(resp.text);
-            expect($("h1.govuk-heading-l").text()).toMatch(HEADING);
+            expect($("h1.govuk-heading-l").text()).toMatch(START_HEADING);
         });
 
         it("should render the start page for /start URL", async () => {
             const resp = await request(app).get(PrefixedUrls.START);
 
             expect(resp.status).toBe(HttpStatusCode.Ok);
             const $ = cheerio.load(resp.text);
-            expect($("h1.govuk-heading-l").text()).toMatch(HEADING);
+            expect($("h1.govuk-heading-l").text()).toMatch(START_HEADING);
         });
     });
+
+    describe("GET method when Planned Maintenance IS set", () => {
+        it("should render the service unavailable page for service root URL", async () => {
+            const plannedMaintenanceOutOfService = { ...plannedMaintenanceResponseUp };
+            plannedMaintenanceOutOfService.resource!.status = "OUT_OF_SERVICE";
+            mockCheckPlannedMaintenance.mockResolvedValue(plannedMaintenanceOutOfService);
+            const resp = await request(app).get(servicePathPrefix);
+
+            expect(resp.status).toBe(HttpStatusCode.NotFound);
+            const $ = cheerio.load(resp.text);
+            expect($("h1.govuk-heading-l").text()).toMatch(SERVICE_UNAVAILABLE_HEADING);
+        });
+
+        it("should render the service unavailable page for /start URL", async () => {
+            const resp = await request(app).get(PrefixedUrls.START);
+
+            expect(resp.status).toBe(HttpStatusCode.NotFound);
+            const $ = cheerio.load(resp.text);
+            expect($("h1.govuk-heading-l").text()).toMatch(SERVICE_UNAVAILABLE_HEADING);
+        });
+    });
+
```

Comment:
Tests for different message formatting of end time?
- zero minutes past hour
- non-zero minutes past hour
(in addition to added tests in date.unit.ts perhaps?)

Reply:
added both into the `start.int` test